Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -3016,6 +3016,32 @@ A note on the flags controlling which atoms/bonds are modified:
   If there is no chiral flag set (i.e. the property is not present), the
   molecule will not be modified.)DOC");
 
+    python::def(
+        "AddStereoAnnotations", Chirality::addStereoAnnotations,
+        (python::arg("mol"), python::arg("absLabel") = "abs ({cip})",
+         python::arg("orLabel") = "or{id}", python::arg("andLabel") = "and{id}",
+         python::arg("cipLabel") = "({cip})",
+         python::arg("cipLabel") = "({cip})"),
```

Comment:
I think the last arg should be `bondLabel`
`std::string cipLabel = "({cip})", std::string bondLabel = "({cip})")`

Reply:
Great catch. I will fix that while handling the merge conflicts with #7155